.gitignore

```diff
@@ -11,3 +11,5 @@
 /tmp-embed/
 /RPMS/
 /crc
+# GoLand metadata
+/.idea
```

Comment:
We usually don't add editor specific configuration folder to ignore list, this should be part of your global git ignore config.

Reply:
Alright then! Closing this